azure-pipelines.yml

```diff
@@ -29,9 +29,11 @@ variables:
   - name: CredFile
     value: appsettings.json
   - name: CredFileLibrary
-    value: product-readiness.omf_api_dotnet.ocs.json
+    value: product-readiness.omf-basic_api-dotnet.ocs.json
   - name: CredFileLibraryOP
```

Comment:
Where possible I have been renaming these to `CredFileLibraryPI` (PI, instead of OnPrem)

Reply:
I will change this to conform with that change

Reply:
Thanks for making that change btw. That one being different than the others was bothering me